app/assets/javascripts/discourse/app/lib/user-status-on-autocomplete.js

```diff
@@ -0,0 +1,29 @@
+import createUserStatusMessage from "discourse/lib/user-status-message";
+
+export function initUserStatusHtml(users) {
+  let instances = [];
```

Comment:
Can be `const`. Also the name is not very descriptive, maybe let's better use `tippyInstances`? (see this comment regarding naming https://github.com/discourse/discourse/pull/21125#issuecomment-1613158817)